nextjs/next1/pages/_app.js

```diff
@@ -0,0 +1,30 @@
+if (process.browser) {
+  //bolt on some next internals that cannot be shared via MF api
+  //yeah it sucks
+  Object.assign(__webpack_share_scopes__.default, {
+    "next/link": {
```

Comment:
next internals cannot be shared through the plugin and have to be added on by the host itself. This only needs to be done once since the webpack global will propagate share scopes when importing federated modules.